arm/Microsoft.ServiceBus/namespaces/readme.md

```diff
@@ -318,7 +318,7 @@ userAssignedIdentities: {
 module namespaces './Microsoft.ServiceBus/namespaces/deploy.bicep' = {
   name: '${uniqueString(deployment().name)}-namespaces'
   params: {
-  
+
```

Comment:
I have to double check these changes for deployment examples as they are the result of a script

Reply:
Jap, this fails unfortunately.

https://github.com/Azure/ResourceModules/runs/7058646082?check_suite_focus=true

Reply:
I slightly modified the logic now to mimic this behavior and will open a PR into your PR soon